src/query/api/v1/handler/prometheus/remote/attribution.go

```diff
@@ -73,8 +89,17 @@ func newPromAttributionMetrics(scope tally.Scope, opts *instrument.AttributionCo
 		return nil, errNoOption
 	}
 	baseScope := scope.SubScope("attribution").SubScope(opts.Name)
+	filters := map[string]string{}
+	for _, filter := range opts.Filters {
+		parts := strings.Split(filter, "=")
+		if len(parts) != 2 {
+			return nil, errInvalidFilter
+		}
+		filters[parts[0]] = parts[1]
```

Comment:
What if the key `parts[0]` is already in the map? Is overwriting fine?

Reply:
good callout, i think override is fine and it is mostly used by us internally.